src/AuthDev/AuthDev.js

```diff
@@ -8,6 +8,7 @@ function setDefaultUser() {
   authData = {
     userId: 1,
     userName: 'le dev',
+    roles: ['user.read'],
```

Comment:
Shouldn't we use `Organization.User` as a default role for AuthDev ?

Reply:
`Organization.Viewer` cause this role has no permission instead of `Organization.User`